executor/linux/build_test.go

```diff
@@ -1432,9 +1453,18 @@ func TestLinux_ExecBuild(t *testing.T) {
 			// Docker uses _ while Kubernetes uses -
 			_pipeline = _pipeline.Sanitize(test.runtime)
 
-			var _runtime runtime.Engine
+			var (
+				_runtime runtime.Engine
+				_pod     *v1.Pod
+			)
```

Comment:
Unlike the other tests, this test will need access to _pod after initialize the k8s runtime.